app/js/referenceSearch.controller.js

```diff
@@ -139,11 +123,9 @@ angular
                 openmrsRest.listFull('conceptreferenceterm', {q: vm.query, limit: vm.entriesPerPage, includeAll: true}).then(function (firstResponse) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2321102930/issues/source?bid=3196969&fileBranchId=3199483#l123)